gleam_stdlib/src/gleam/int.gleam

```diff
@@ -16,3 +16,10 @@ pub fn compare(a, b) {
     }
   }
 }
+
+pub fn min(a, b) {
+  case a < b {
+  | True -> a
+  | False -> b
+  }
+}
```

Comment:
Could we have a newline at the end of this file please?

Reply:
done